securedrop/source_templates/index.html

```diff
@@ -3,9 +3,9 @@
 
 <head>
   {% if g.organization_name == "SecureDrop" %}
-  <title>{{ g.organization_name }} | {{ gettext('Protecting Journalists and Sources') }}</title>
+  <title>{{ gettext('Portal') }} | {{ g.organization_name }} | {{ gettext('Protecting Journalists and Sources') }}</title>
   {% else %}
-  <title>{{ g.organization_name }} | {{ gettext('SecureDrop') }}</title>
+  <title>{{ gettext('Portal') }} | {{ g.organization_name }} | {{ gettext('SecureDrop') }}</title>
```

Comment:
`Portal`? Can you give a little bit of context on this addition @saffronsnail? (Apologies if I'm missing any previous conversation on the topic!)

Reply:
I'm very open to suggestions about a better title for this page. Most of the pages have a descriptive built-in header, but the landing page for the source interface has 2 sections, one for creating a new account and one for logging in to an existing account. I feel like titling the page "Log In" is not technically accurate because creating a new account is a distinct action from logging in, but I might be over-thinking it. For reference, the rendered page can be seen in the attached screenshot.
![source-index](https://user-images.githubusercontent.com/8052449/212785986-500568be-2b39-4685-8bc0-60827ae6899c.png)

Reply:
I think you've got a good point. And "Log in or sign up" doesn't really do it because "sign up" probably gives the wrong idea. "Portal" isn't bad. :+1: